.travis.yml

```diff
@@ -4,16 +4,21 @@ sudo: false
 
 language: php
 
+env:
+  global:
+    - CORE_RELEASE=master
```

Comment:
CORE_RELEASE is always the tag / branch name.

When we create a `4` branch then it'll be 4.